### PR TITLE
JSDK-2641 set mediaEl.srcObject only if needed

### PR DESCRIPTION
**Pull request by makarandp0**
We were setting srcObject to workaround a [chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=720258) - which is fixed now. This change avoids the flicker.  

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Comment by timmydoza** [comment]
Looks good to me! The Ahoy App still has some flicker, but it's because of a React issue.  Tomorrow I'll try out the React fix with this version of the SDK and see how it goes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/suppor
 Bug Fixes
 ---------
 
+- Fixed a bug where calling `VideoTrack.attach` with `video` element that was attached to the DOM caused the video to flicker. (JSDK-2641)
+
 - Fixed a bug where switching between networks (or connecting to VPN) sometimes caused media flow to stop. (JSDK-2667)
+
```
Comment (manjeshbhargav) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,10 @@ For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/suppor
 Bug Fixes
 ---------
 
+- Fixed a bug where calling `VideoTrack.attach` with `video` element that was attached to the DOM caused the video to flicker. (JSDK-2641)
+
```
Comment (manjeshbhargav) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,10 @@ For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/suppor
 Bug Fixes
 ---------
 
+- Fixed a bug where calling `VideoTrack.attach` with `video` element that was attached to the DOM caused the video to flicker. (JSDK-2641)
```
Comment (manjeshbhargav) [inline CHANGELOG.md]:
```suggestion
- Fixed a bug where attaching a VideoTrack to a HTMLVideoElement that was already in the DOM caused the video to flicker. (JSDK-2641)
```

**Pull request merged**